dashboard/app/controllers/javabuilder_sessions_controller.rb

```diff
@@ -13,40 +13,105 @@ class JavabuilderSessionsController < ApplicationController
 
   # GET /javabuilder/access_token
   def get_access_token
-    teacher_list = get_teacher_list.join(',')
+    begin
+      require_default_params
+      params.require(:channelId)
```

Comment:
thought on reducing duplicate code: add a parameter to `require_default_params` for additional params to add to the list

Reply:
I can do that 👍